c/datatable.cc

```diff
@@ -145,25 +145,25 @@ void DataTable::delete_all() {
 }
 
 
+
 // Split all columns into groups, by their `RowIndex`es
-static std::pair<std::vector<RowIndex>, std::vector<std::vector<size_t>>>
-_split_columns_by_rowindices(const DataTable* dt)
-{
-  std::vector<RowIndex> rowindices;
-  std::vector<std::vector<size_t>> colindices;
-  for (size_t i = 0; i < dt->ncols; ++i) {
-    RowIndex r = dt->columns[i]->rowindex();
-    size_t j = 0;
-    for (; j < rowindices.size(); ++j) {
-      if (rowindices[j] == r) break;
+std::vector<RowColIndex> DataTable::split_columns_by_rowindices() const {
```

Comment:
May be a better name for this method is `group_columns_by_rowindices()`?

Reply:
sure